DEPLOYMENT_GUIDE.md

```
@@ -29,7 +29,7 @@ REACT_APP_API_BASE_URL=http://localhost:8000
 ### 3. Deploy with Docker Compose
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)